content/src/main/content/META-INF/vault/filter.xml

```diff
@@ -48,4 +48,5 @@
         <include pattern="/etc/acs-commons/workflow-remover(.*)"/>
     </filter>
     <filter root="/apps/cq/core/content/nav/tools/acs-commons"/>
+    <filter root="/apps/cq/gui/components/authoring/dialogsitewide"/>
```

Comment:
Moved to /apps/acs-commons

This is now refactored on the HS2 repo. Once we finalize our updates I'll update this PR.